internal/core/config/config.go

```diff
@@ -86,6 +86,8 @@ type Config struct {
 	MetricsToExclude []MetricFilter `yaml:"metricsToExclude" default:"[]"`
 	// A list of properties filters
 	PropertiesToExclude []PropertyFilterConfig `yaml:"propertiesToExclude" default:"[]"`
+	// Whether to enable new-style metrics filter
+	EnableExtraMetricsFilter bool `yaml:"enableExtraMetricsFilter" default:"false"`
```

Comment:
Other name ideas:
 - `enableStandardMetricFilters`
 - `enableStandardFilters`
 - `enableCustomMetricFilters`
 - `enableWhitelistFilters`

Main thing against "extra" is that it doesn't correspond to anything we document about how host based pricing works.
 - `